src/config/database.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+    database: 'mongodb://localhost:27017/tageler_db'
```

Comment:
We need to config to stay overridable through env for the docker deploy to be able to find it's database.

The infos need to come from the `process.env.MONGODB_PORT_27017_TCP_ADDR` (should default to 'localhost') and `process.env.MONGODB_DATABASE_NAME` (should default to 'tageler_db') env variables.

The original call to mongoose.connect looked as follows:
```js
mongoose.connect(
  process.env.MONGODB_PORT_27017_TCP_ADDR || 'localhost',
  process.env.MONGODB_DATABASE_NAME || 'tageler'
);
```